src/components/BusinessCaseReview/index.tsx

```diff
@@ -1,106 +1,42 @@
-import React, { useState } from 'react';
+import React from 'react';
 
-import ReviewRow from 'components/ReviewRow';
-import {
-  DescriptionDefinition,
-  DescriptionList,
-  DescriptionTerm
-} from 'components/shared/DescriptionGroup';
-import ResponsiveTabs from 'components/shared/ResponsiveTabs';
-import { hasAlternativeB } from 'data/businessCase';
 import { BusinessCaseModel } from 'types/businessCase';
-import AsIsSolutionReview from 'views/BusinessCase/Review/AsIsSolutionReview';
-import ProposedBusinessCaseSolutionReview from 'views/BusinessCase/Review/ProposedBusinessCaseSolutionReview';
+
+import AlternativeAnalysisReview from './AlternativeAnalysisReview';
+import GeneralRequestInfoReview from './GeneralRequestInfoReview';
+import RequestDescriptionReview from './RequestDescriptionReview';
 
 type BusinessCaseReviewProps = {
   values: BusinessCaseModel;
 };
 
 const BusinessCaseReview = ({ values }: BusinessCaseReviewProps) => {
-  const [activeSolutionTab, setActiveSolutionTab] = useState(
-    '"As is" solution'
-  );
-
-  const getFilledSolutions = () => {
-    const solutions = [
-      '"As is" solution',
-      'Preferred solution',
-      'Alternative A'
-    ];
-    if (hasAlternativeB(values.alternativeB)) {
-      solutions.push('Alternative B');
-    }
-    return solutions;
-  };
-
   return (
     <>
       <div className="grid-container">
         <h2 className="font-heading-xl">General request information</h2>
-        <DescriptionList title="General request information">
-          <ReviewRow>
-            <div>
-              <DescriptionTerm term="Request Name" />
-              <DescriptionDefinition definition={values.requestName} />
-            </div>
-            <div>
-              <DescriptionTerm term="Business Owner" />
-              <DescriptionDefinition definition={values.businessOwner.name} />
-            </div>
-          </ReviewRow>
-          <ReviewRow>
-            <div>
-              <DescriptionTerm term="Requester" />
-              <DescriptionDefinition definition={values.requester.name} />
-            </div>
-            <div>
-              <DescriptionTerm term="Requester Phone Number" />
-              <DescriptionDefinition
-                definition={values.requester.phoneNumber}
-              />
-            </div>
-          </ReviewRow>
-        </DescriptionList>
+        <GeneralRequestInfoReview
+          values={{
```

Comment:
I have a hunch we copied the values pattern here from our form components. But it seems a bit of an odd parameters pattern now that this has been broken out. Could we send the values as the `businessCase` (just a rename), or even potentially pass the values as their own named parameters, instead of one large object?